### PR TITLE
TDD Project

### PR DESCRIPTION
**Pull request by Bhekisile**
## Changes

-  Wrote tests for all the methods and then created the class and the methods below.
-  Created a class called Solver.
-  Created a method called factorial that takes one argument, an integer N, and returns the [factorial]
-  Created a method called reverse that takes one argument, a string word, and returns word reversed
-  Created a method called fizzbuzz that takes one argument, an integer N, and returns a string.

### CODE REVIEW

File: MIT.md [hunk 1]
```diff
@@ -1,10 +1,7 @@
-## Copyright 2021, [YOUR NAME]
+## Copyright 2021, [Bhekisile & Akorede]
```
Comment (Shedrack-Sunday) [inline MIT.md]: - [ ] Kindly correct the year in your license file 👍

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @Bhekisile and @Akorede3133         👋 

# CHANGES REQUIRED ♻️

Impressive work on the project so far👏
You've done extremely well so far in this project,  However, there are some adjustments you need to make to move to the next project. 💯



## Highlights   
- Good use of GitHub features ✔️   
- Nice work with the Ruby features ✔️
- Good job with class calls and testing    ✔️  

# - Required Changes  🌵  

- [x] Please make sure to cross-check all sections to avoid wrong descriptions in your documentation


_Check the comment below




# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing really improves your application.



Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: MIT.md (hunk 1 above)
Comment (Bhekisile) [reply MIT.md]: Fixed. Thank you

**Review by iambenkis — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

Good job with class calls and testing ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @iambenkis in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**